Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -1766,8 +1766,16 @@ IntRect FrameSelection::absoluteCaretBounds(bool* insideFixed)
 
 static void repaintCaretForLocalRect(Node* node, const LayoutRect& rect, CaretAnimator* caretAnimator)
 {
-    if (auto* caretPainter = rendererForCaretPainting(node))
-        caretPainter->repaintRectangle(caretAnimator ? caretAnimator->repaintCaretRectForLocalRect(rect) : rect);
+    constexpr bool shouldClipToLayer = false;
```

Comment:
These could move down into the clause that uses them. And if `repaintRectangle` took the enum values, you wouldn't need them at all.